src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String COMMA = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            for (String value = bufferedReader.readLine();
+                    value != null; value = bufferedReader.readLine()) {
+                String[] array = value.split(COMMA);
+                if (value.contains(SUPPLY)) {
```

Comment:
as well, it's better to use value[OPERATION_INDEX].equals(SUPPLY)) it's gonna be easier to work with this code if we have more operations with similar name (this is not obligatory)